test_lua/helpers/cluster2.lua

```diff
@@ -6,13 +6,13 @@ local fiber = require('fiber')
 local cluster = {}
```

Comment:
I'm a bit confused. Where is the `cluster1` then?

Reply:
No idea, it was named `cluster2` before me 🙃